src/screens/OtherProfile/index.js

```diff
@@ -311,7 +232,7 @@ const OtherProfile = () => {
 
   const __renderListHeader = () => {
     const __renderBlockIcon = () => {
-      if (blockStatus.blocker)
+      if (isBlocking)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `blockStatus.blocker` has been replaced with `isBlocking`. Ensure that `isBlocking` provides the same functionality as `blockStatus.blocker`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->